src/press/models.py

```diff
@@ -441,20 +428,117 @@ def navigation_items_for_sitemap(self):
+            )
+        )
+        return journals.order_by("journal_name")
+
+    def apply_journal_ordering(self, journals):
+        if self.order_journals_az:
+            return self.apply_journal_ordering_az(journals)
+        else:
+            # Journals will already have been ordered according to Meta.ordering
+            return journals
+
+    @property
+    def journals_az(self):
+        """
+        Deprecated. Use `journals` with Press.order_journals_az turned on.
+        """
+        warnings.warn(
+            "`Press.journals_az` is deprecated. "
+            "Use `Press.journals` with Press.order_journals_az turned on."
+        )
+        Journal = apps.get_model("journal", "Journal")
+        return self.apply_journal_ordering_az(Journal.objects.all())
+
     @property
     def public_journals(self):
-        Journal = apps.get_model("journal.Journal")
+        """
+        Get all journals that are not hidden from the press
+        or designated as conferences.
+        Do not apply ordering yet,
+        since the caller may filter the queryset.
+        """
+        Journal = apps.get_model("journal", "Journal")
         return Journal.objects.filter(
             hide_from_press=False,
             is_conference=False,
-        ).order_by("sequence")
+        )
+
+    @property
+    def public_active_journals(self):
+        """
+        Get all journals that are visible to the press
+        and marked as 'Active' or 'Test' in the publishing status field.
+
+        Note: Test journals are included so that users can test the journal
+        list safely. A separate mechanism exists to hide them from the press
+        once the press enters normal operation:
+        Journal.hide_from_press.
+        """
+        Journal = apps.get_model("journal.Journal")
+        return self.apply_journal_ordering(
+            self.public_journals.filter(
+                status__in=[
+                    Journal.PublishingStatus.ACTIVE,
+                    Journal.PublishingStatus.TEST,
+                ]
+            )
+        )
+
+    @property
+    def public_archived_journals(self):
+        """
+        Get all journals that are visible to the press
+        and marked as 'Archived' in the publishing status field.
+        """
+        Journal = apps.get_model("journal.Journal")
+        return self.apply_journal_ordering(
+            self.public_journals.filter(
+                status=Journal.PublishingStatus.ARCHIVED,
+            )
+        )
+
+    @property
+    def public_coming_soon_journals(self):
+        """
+        Get all journals that are visible to the press
+        and marked as 'Coming soon' in the publishing status field.
+        """
+        Journal = apps.get_model("journal.Journal")
+        return self.apply_journal_ordering(
+            self.public_journals.filter(
+                status=Journal.PublishingStatus.COMING_SOON,
+            )
+        )
```

Comment:
These sound more like either individual custom managers or a single custom manager with multiple methods. The rule of thumb is to ask oneself, do my model instances need to invoke this property or is this a special way of querying model-instances? If the answer is the latter, it is a good indication that you are writing a Manager method rather than a Model property.